config/config.default.js

```diff
@@ -17,9 +17,11 @@ module.exports = () => {
    * @property {Object} xssProtection - 是否开启IE8的XSS Filter，默认开启。
    * @property {Object} csp - csp安全配置。
    * @property {Array} domainWhiteList - 安全跳转白名单
+   * @property {Array} protocolWhiteList - 协议白名单
    */
   exports.security = {
     domainWhiteList: [],
+    protocolWhiteList: [],
```

Comment:
If bringing break change is tolerated in this period of time, I'm ok with that.

Publish a major version in egg 1.0?